### ISSUE TITLE
Album Art Appears But Only Some Artist Art?

### ISSUE THREAD

**Issue by 0pLuS0** [gateship-one/odyssey#203]
Hello,

### Version of Odyssey
1.1.19

### Android Version
OxygenOS 9.0.11

### Device Manufacturer and model
OnePlus 5T

I don't listen to popular mainstream bands, by the typical definition of ```Pop Music```. So I assumed because of the music I listen to, this is the problem.

What I noticed is that with some of the bands I listen to, the Album Art appeared in Odyssey, but out of 4 albums, only one artist image appeared.

I would assume that if the Album Art can work, it only makes sense that I can see Artist Art as well for all the bands.

I'm I'm understanding the settings correctly, only ```Fanart.tv``` is used for the Artist Images? If this is true, hopefully this can change in a future version to also include other sources, I thought that MusicBrainz and LastFM were going to provide for these images as well?

```Notice the screen shots below, how the Album Art appears for all the Artists, but not the Artist Images```... 

![albums](https://user-images.githubusercontent.com/36190223/89723059-fa180000-d9e0-11ea-8d27-8bb93cbe5b16.jpg)

![artists](https://user-images.githubusercontent.com/36190223/89723062-ff754a80-d9e0-11ea-9a8a-1617ecf3b61d.jpg)

```P.S. In the meantime, for the Artist Images that are not appearing, is there a way I can do this from a local folder with odyssey?```

THANKS

**Comment by 0pLuS0** [comment]
Is anyone around?

I was hoping to get a reply in a few days to try and solve this...

THANKS

**Comment by gnome17** [comment]
Yes we are reading the issues that pop up here, but we are maintaining this project in our free time and sometimes there is just no time left for a proper answer.

We are trying to answer issues quickly if we can provide a quick solution for the problem which is unfortunately not the case for your issue.

AFAIK MusicBrainz has no support for Artist images. In the past we included last.fm as an additional artist provider but they stopped to provide valid artist images some time ago.

We already described in other issues why we don't support local artist images. With more and more restrictions regarding file access from google, we are not even sure if we can provide local album images in future android versions.

There is already an open issue regarding adding discogs for MALP (https://gitlab.com/gateship-one/malp/-/issues/166).

So far we didn't have time to look into it and currently we don't have that many time left for both apps.

So long story short if Fanart.tv doesn't provide images for your artist you will not get any in odyssey. Sorry.

**Comment by 0pLuS0** [comment]
Hi,

I assumed if St. Vincent was showing up, then the other Artist Images would appear in that bottom screen shot.

I don't have the app installed, I reflashed my phone with a different ROM.

What site are the Artist Images being pulled from? Maybe I can look online to see if I can find any image for them, in fact I thought I had a look when I had Odyssey installed and that I did notice some.

Thanks

**Comment by gnome17** [comment]
Hi,
actually there is not really something to fix because it should work as intended. If we don't find an artist image via Fanart.tv we won't show an artist image.

As I said before we maybe add more providers in the future but we will not promise that.

Of course there is still the possibility that our look up has still a bug or your android device messed up reading the tags properly but at the moment it seems that it is more likely that there are just no images for your artists.

If you provide me with the tags for one of your music tracks I could try to verify that.

**Comment by 0pLuS0** [comment]
Provide what in the way of tags?

**Comment by gnome17** [comment]
What I meant was the id3v2 tags from one of your songs of an album that has issues loading a cover for album or artist.
But this was just to verify if Odyssey has a bug but as I said before it is more likely that the used image provider don't have images for your album/artist.

**Comment by 0pLuS0** [comment]
Ok
